olive/strategy/search_parameter.py

```diff
@@ -80,8 +80,8 @@ class Boolean(Categorical):
     >>> Boolean()
     """
 
-    def __init__(self):
-        super().__init__([True, False])
+    def __init__(self, support: List[bool] = [True, False]):
```

Comment:
this has the same issue as using mutable objects as default values. 

if we only want one option, we can just use `Categorical([False])` instead. `Boolean` was created as a shorthand for `Categorical([True, False])`

Reply:
Fixed.